packages/drivers/local-driver/src/localDocumentServiceFactory.ts

```diff
@@ -48,7 +47,9 @@ export class LocalDocumentServiceFactory implements IDocumentServiceFactory {
         clientIsSummarizer?: boolean,
     ): Promise<IDocumentService> {
         ensureFluidResolvedUrl(resolvedUrl);
-        assert(!!createNewSummary, 0x202 /* "create empty file not supported" */);
+        if (createNewSummary === undefined) {
+            throw new Error("Empty file summary creation isn't supported in this driver.");
```

Comment:
@ssimic2 - What was your follow-up about UsageError?  Whatever we decide from [here](https://github.com/microsoft/FluidFramework/pull/9650/files#r859003637) will also apply here I expect.

@justus-camp - This is good for now, but we might end up coming back later to change this to an error with a specific errorType (likely "usageError")

Reply:
Sounds good. I thought about implementing a new usageError type but used the generic for now. We can circle back around and add usageError if we find it necessary